File: FFRv2/app/src/main/java/com/devingotaswitch/rankings/domain/projections/DefensiveProjection.java
package com.devingotaswitch.rankings.domain.projections;

import com.devingotaswitch.rankings.domain.ScoringSettings;

class DefensiveProjection extends ProjectionBase {
    private final double projection;

    DefensiveProjection(double projection) {
        this.projection = projection;
    }

    @Override
    double getProjectedPoints(ScoringSettings scoringSettings) {
        return projection;
    }

    @Override
    String getDisplayString() {
        return "";
    }
}

File: FFRv2/app/src/main/java/com/devingotaswitch/rankings/domain/projections/DefensiveProjection.kt
package com.devingotaswitch.rankings.domain.projections

import com.devingotaswitch.rankings.domain.ScoringSettings

internal class DefensiveProjection(private val projection: Double) : ProjectionBase() {
    override fun getProjectedPoints(scoringSettings: ScoringSettings): Double {
        return projection
    }

    override fun getDisplayString(): String {
        return ""
    }

}